scripts/install-mender.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/sh
+
+set -e
+
+CHANNEL="stable"
+
+DEFAULT_COMPONENTS="mender-client"
+COMPONENTS="$DEFAULT_COMPONENTS mender-shell"
+SELECTED_COMPONENTS=$DEFAULT_COMPONENTS
+
+export DEBIAN_FRONTEND=noninteractive
+
+banner (){
+    echo "
+                          _
+ _ __ ___   ___ _ __   __| | ___ _ __ 
+| '_ \` _ \ / _ \ '_ \ / _\` |/ _ \ '__|
+| | | | | |  __/ | | | (_| |  __/ |   
+|_| |_| |_|\___|_| |_|\__,_|\___|_|   
+
+Running the Mender installation script.
+--
+"
+                                      
+}
+
+usage() {
+    echo "usage: install-mender.sh [-h] [-c channel] [component...]"
+    echo "  -h             print this help"
+    echo "  -c             channel: stable(default)|experimental"
+    echo "  <component>    list of components to install"
+    echo ""
+    echo "Supported components:"
+    echo "  * mender-client (default if none provided)"
+    echo "  * mender-shell"
+}
+
+is_known_component() {
+    for known in $COMPONENTS
+    do
+	if [ "$1" = "$known" ]	
+ 	then
+	  return 0 
+	fi
+    done
+    return 1
```

Comment:
Generally, in the file, there is a mix of indentation, switching between two and four spaces. Would be nice if it was cleaned up (I think four spaces it what we use in most other files).

Reply:
argh, yeah, sorry - will fix.

it's because in part I'm editing the file in an `rw` mounted docker file :)  but the vim there is rudimentary and has default tab/space expansion settings.